sdcm/nemesis.py

```diff
@@ -161,7 +161,6 @@ def __init__(self, tester_obj, termination_event, *args):  # pylint: disable=unu
         self._add_drop_column_columns_info = {}
         self._add_drop_column_target_table = []
         self._add_drop_column_tables_to_ignore = {
-            'scylla_bench': '*',  # Ignore scylla-bench tables
```

Comment:
Why do you remove it?

Reply:
It is all in commit message, new version of s-b supports column adding/removing